src/components/interaction-designer/flow-editors/import/ErrorHandler.vue

```diff
@@ -7,7 +7,7 @@
       </div>
```

Comment:
I know the above line was in a previous PR, but I wondering if you can take the opportunity to localize this missed text.

For info: in case of TS error related to `trans()`, check [here](https://github.com/FLOIP/flow-builder/blob/master/src/components/interaction-designer/block-types/ConsoleIO_PrintBlock.vue#L58) how I load the `Lang` mixin within class based component, pay attention to the importation `import Lang from '@/lib/filters/lang'` instead of ` { lang } `

Reply:
Thanks for making changes here as well